aws/nextflow-compute.tf

```diff
@@ -0,0 +1,56 @@
+resource "aws_iam_instance_profile" "nf_ecs_instance_role" {
+  name = "nextflow-ecs-instance-role"
+  role = aws_iam_role.nf_ecs_role.name
+}
+
+resource "aws_batch_compute_environment" "nextflow_ondemand" {
+  compute_environment_name = "nextflow-ondemand-compute"
+  compute_resources {
+    instance_role = aws_iam_instance_profile.nf_ecs_instance_role.arn
+    instance_type = [
+      "optimal",
+    ]
+    allocation_strategy = "BEST_FIT"
+    max_vcpus = 20
+    min_vcpus = 0
+    image_id = "ami-0a8857ac38c35157f"
```

Comment:
I know you noted where this AMI came from and what it is in the PR description, but I think it'd be good to also note that in a comment here.

Reply:
Oh I just saw `aws/setup-log.md`, maybe just mention that doc in a comment here?

Reply:
Added.